SPECS/core-packages/core-packages.spec

```diff
@@ -62,13 +62,9 @@ Requires:       expat
 Requires:       filesystem
 Requires:       findutils
 Requires:       mariner-release
-Requires:       mariner-repos-preview
-Requires:       mariner-repos-extended-preview
```

Comment:
Just to clarify- we don't want the extended repo in our core package set? It makes sense to me, but I just want to double check.

Reply:
I was going to keep them out for now.  Force everyone to add them if they want them.  There is no CVE SLA there.